### DISCUSSION TITLE
请求支援亿安云

### DISCUSSION THREAD

**Discussion by fengguowudi** [AlistGo/alist#500] [Ideas]
前面两次不懂事，没先看有无重复，感到十分抱歉！
https://act.i95cloud.com/
这是网盘网址，速度还是不错的，而且目前可以换ip无限领会员~~~~

**Comment by fengguowudi** [comment]
同时我有个小疑问，extract_folder 这个是啥意思啊，翻译和百度都没看懂。。

**Reply by xhofe** [reply]
排序用的
front：所有文件夹提取到最前面
back：最后面

**Comment by xhofe** [comment]
这不就是cloudreve套娃OneDrive吗。。。

**Comment by fengguowudi** [comment]
好家伙。。。我错了，下次精心挑选再来，告辞

**Comment by kadong1** [comment]
亿安的话，他可以直接用wedDav挂

**Reply by kadong1** [reply]
我好多网盘都是这么挂的。

**Reply by youtehub** [reply]
可以说一下你使用了哪些网盘，是直接支持 wedDav 挂载的吗，目前只知道亿安云，阿里云盘

**Reply by youtehub** [reply]
后面也想转到这些网盘去

**Reply by kadong1** [reply]
> 后面也想转到这些网盘去

坚果云 绮梦网盘 TeraCLOUD  PowerFolder

**Reply by kadong1** [reply]
> 后面也想转到这些网盘去

诚通

**Reply by kadong1** [reply]
> > 后面也想转到这些网盘去
> 
> 诚通

城通的共有云和私有云都可以挂载

**Reply by youtehub** [reply]
好的，已经收录到博客中了，后面慢慢更换
